crates/js-sys/src/lib.rs

```diff
@@ -5528,6 +5528,65 @@ pub mod Intl {
             )
         }
     }
+
+    // Intl.RelativeTimeFormat
+    #[wasm_bindgen]
+    extern "C" {
+        /// The `Intl.RelativeTimeFormat` object is a constructor for objects
+        /// that enable language-sensitive relative time formatting.
+        ///
+        /// [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat)
+        #[wasm_bindgen(extends = Object, js_namespace = Intl, typescript_type = "Intl.RelativeTimeFormat")]
+        #[derive(Clone, Debug)]
+        pub type RelativeTimeFormat;
+
+        /// The `Intl.RelativeTimeFormat` object is a constructor for objects
+        /// that enable language-sensitive relative time formatting.
+        ///
+        /// [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat)
+        #[wasm_bindgen(constructor, js_namespace = Intl)]
+        pub fn new(locales: &Array, options: &Object) -> RelativeTimeFormat;
+
+        /// The `Intl.RelativeTimeFormat.prototype.format` property returns a getter function that
```

Comment:
```suggestion
        /// The `Intl.RelativeTimeFormat.prototype.format` property
```

The formatted string is returned directly; no getter function is involved.

Reply:
Good catch! Forgot to change this after removing getter. And its no longer a property ether. I'll change it.

Reply:
Corrected in [43d041e](https://github.com/rustwasm/wasm-bindgen/pull/3182/commits/43d041e7303f8602429d1c2aaa39495d8767502c).